backend/src/main/java/ca/bc/gov/app/exception/InvalidRequestObjectException.java

```diff
@@ -6,7 +6,11 @@
 
 @ResponseStatus(value = HttpStatus.BAD_REQUEST)
 public class InvalidRequestObjectException extends ResponseStatusException {
+
+  private static final long serialVersionUID = -2996185412447039174L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data